docs/mp/metrics/metrics-capable-components.adoc

```diff
@@ -17,6 +17,7 @@
 ///////////////////////////////////////////////////////////////////////////////
 
 = Metrics-Capable Modules
+:h1-prefix: MP
 :description: Helidon MP metrics-capable modules
 :keywords: helidon, metrics, metrics-capable, microprofile, guide
 :intro-project-name: {h1-prefix}
```

Comment:
How about moving `:intro-project-name: {h1-prefix}` after the include of `mp.adoc` ; does that work ?

Reply:
Another way, which I think is better, is to use conditionals in the common content directly.

This makes the common docs "polymorphic" (big word :D) ; and sort of standalone (as it depends on the flavor include).

It does simplify the "contract" of these kind of common documents.
It can also help the GitHub rendering and set default values for the cases where the required attributes are not set.